WordPress/Classes/ViewRelated/Notifications/ReplyTextView/ReplyTextView.swift

```diff
@@ -104,6 +98,16 @@ import WordPressShared.WPStyleGuide
         textView.replace(newRange, withText: replacementText)
     }
 
+    @objc open func collapseReplyTextView() {
+        isExpanded = false
+        self.headerView.isHidden = true
+        UIView.animate(withDuration: 0.5, animations: {
```

Comment:
You're using `0.5` in quite a few places. It would be preferable to define this as a constant somewhere rather than using the same literal value in various places, as it makes it harder to typo it in one place and easier to update if you want to tweak the entire animation speed.